### PR TITLE
Add /pt-br/infrastructure-as-a-service.md

### LINKED ISSUE #827: [Portuguese] Localize 'Infrastructure as a Service'
Localize https://glossary.cncf.io/infrastructure-as-a-service/

### PR DESCRIPTION
**Pull request by MrErlison**
Signed-off-by: Mr. Erlison <sisal.cordame_0a@icloud.com>

Added translation for portuguese of the term infrastructure as a service.

resolves #827

- [x] This PR does not contain plagiarism
- [x] I have signed off on all commits

### CODE REVIEW

File: content/pt-br/infrastructure-as-a-service.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: Infraestrutura como Serviço (IaaS)
+status: Completed
+category: tecnologia
+tags: ["infraestrutura", "", ""]
+---
+
+## Como é
+
+Infraestrutura como serviço, ou IaaS, é um modelo de serviço de [computação em nuvem](/pt-br/cloud-computing/) que oferece máquinas [física](/pt-br/bare-metal-machine/) ou [virtualizada](/pt-br/virtualization/), armazenamento e recursos de rede sob demanda em um modelo pague conforme o uso.
```
Comment (edsoncelio) [inline content/pt-br/infrastructure-as-a-service.md:10]: typo: `máquinas física` -> `máquinas físicas`

File: content/pt-br/infrastructure-as-a-service.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+---
+
+## Como é
+
+Infraestrutura como serviço, ou IaaS, é um modelo de serviço de [computação em nuvem](/pt-br/cloud-computing/) que oferece máquinas [física](/pt-br/bare-metal-machine/) ou [virtualizada](/pt-br/virtualization/), armazenamento e recursos de rede sob demanda em um modelo pague conforme o uso. 
+Os provedores de nuvem possuem e operam o hardware e o software, disponíveis para os consumidores em implantações de nuvem pública, privada ou híbrida.
+
+## Problema relacionado
+
+Em configurações tradicionais (_on-premise_), as organizações geralmente concentram seus enforços no uso efetivo de recursos de computação.
```
Comment (edsoncelio) [inline content/pt-br/infrastructure-as-a-service.md:15]: typo: `enforços` -> `esforços`

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Just a few typos, after that will be LGTM, thanks!

**Review by edsoncelio — APPROVED** [review]
LGTM

**Review by jessicalins — APPROVED** [review]
LGTM, thanks!

**Pull request merged**